schema/enums/QuantitySourceType.schema.json

```diff
@@ -12,5 +12,5 @@
     "INSTRUMENT_MAX",
     "INSTRUMENT_MIN"
   ],
-  "$comment": "Copyright © 2023 Open Cap Table Coalition (https://opencaptablecoalition.com) / Original File: https://github.com/Open-Cap-Table-Coalition/Open-Cap-Format-OCF/tree/main/schema/types/QuantitySourceType.schema.json"
+  "$comment": "Copyright © 2023 Open Cap Table Coalition (https://opencaptablecoalition.com) / Original File: https://github.com/Open-Cap-Table-Coalition/Open-Cap-Format-OCF/tree/main/schema/enums/quantitysourcetype.schema.json"
```

Comment:
This schema is actually in the wrong place too... `https://github.com/Open-Cap-Table-Coalition/Open-Cap-Format-OCF/blob/main/schema/types/QuantitySourceType.schema.json`

It needs to be moved into the enums folder too or various tools wouldn't work properly.